simulator/src/main/java/com/hazelcast/simulator/coordinator/TestCaseRunner.java

```diff
@@ -119,8 +119,8 @@ boolean run() {
             echo("Finished Test global tear down");
 
             echo("Starting Test local tear down");
-            agentsClient.waitForPhaseCompletion(prefix, testCaseId, "localTeardown");
             agentsClient.executeOnAllWorkers(new GenericCommand(testCaseId, "localTeardown"));
+            agentsClient.waitForPhaseCompletion(prefix, testCaseId, "localTeardown");
```

Comment:
I have no clue. But it's just the local teardown, so in the worst case we just didn't destroy some objects if the workers got killed to soon.